data/osx/Cataclysm.sh

```diff
@@ -4,6 +4,7 @@ cd "${PWD}/../Resources/"
 
 COMMAND='
 OSREV=`uname -r | cut -d. -f1`;
+export TERMINFO=/usr/share/terminfo;
```

Comment:
Nothing.

But after I revert all three commits, uninstall libiconv and ncurses, it seems completely ok now. So I guess the problem is caused by libncursesw from homebrew.